### DISCUSSION TITLE
Running dependabot on ARC

### DISCUSSION THREAD

**Discussion by danpowell88** [community/community#160504] [Actions]
I see now github has enabled dependabot on self hosted runners if you added a label to the runners, does this extend out to ARC runners. I don't see any way to add labels (using the newer ARC functionality).

**Comment by galoniz** [comment]
did someone tried it and have it running?

**Comment by da1910** [comment]
I was able to get this running using Runner Scale Sets, using the `dind` container mode. Depending on your Kubernetes environment and image policies you may need to override some of the configuration (for example if you need a custom tag for your `dind` container)

You need to deploy the set with the exact name `dependabot` which means you can only have one, either at repository level or shared across the org.

See https://github.com/actions/actions-runner-controller/discussions/3340

**Reply by galoniz** [reply]
how can you tell it's working? I do not see any mention in the dependabot section regarding the host it runs at, can you please advise?

I have created it with the right label: 
<img width="955" alt="image" src="https://github.com/actions/actions-runner-controller/assets/70090212/06434188-a24a-4ef4-a536-22ca3bb1a4de">

but I do not see any logs on the runner itself

**Reply by da1910** [reply]
Have you activated the setting to allow dependabot to run using Actions infrastructure? It's also possible it doesn't support runner groups (I did not assign dependabot to a group)

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/about-dependabot-on-github-actions-runners

**Reply by phknot** [reply]
@da1910 I hate to necro a dead thread, but this seems to be the only available info online of running Dependabot updates using ARC. I am running into problems where the Dependabot job can't connect to the Docker socket. I have tried both the default `containermode.type: dind`  and the rootless setup from the documentation.

Could you provide a bit more context of your setup? Thanks :)
